### PR TITLE
Is309/webserver uses apihub

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?
Moving openapi specs of webserver to apihub/specs
 - cleanup config of subsystems. Every subsystem injects config schema (see _config.py submodules)
 - cleanup webserver skeleton
 - rest subsystem loads specs from file or url
 - fix openapi envelopes
-  fix servicelib auto-routing

## Related issue number
connected to #309 

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [x] **Runs in the swarm**
- [x] If you design a new module, add your user to .github/CODEOWNERS

[waffle.io]:https://waffle.io/marketing-assets/documents/waffleio_cheatsheet_v1.pdf?utm_source=blog&utm_medium=cheatsheet-ctabutton&utm_campaign=cheatsheet

### CODE REVIEW

File: services/docker-compose.yml [hunk 1]
```diff
@@ -29,8 +29,8 @@ services:
       - POSTGRES_USER=${POSTGRES_USER}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
       - POSTGRES_DB=${POSTGRES_DB}
-      - POSTGRES_HOST=${POSTGRES_HOST}
-      - POSTGRES_PORT=${POSTGRES_PORT}
+      - POSTGRES_HOST=postgres
+      - POSTGRES_PORT=5432
```
Comment (sanderegg) [inline services/docker-compose.yml]: you probably do not want this changes in the main docker-compose right?

File: services/web/server/src/simcore_service_webserver/s3.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    # app["s3.client"] = client
+
+
+# alias
+setup_s3 = setup
+
+__all__ = (
+    'setup_s3'
+)
```
Comment (sanderegg) [inline services/web/server/src/simcore_service_webserver/s3.py]: why do you create a s3 client, is it not suppose to be the storage service role?

File: services/docker-compose.yml (hunk 1 above)
Comment (pcrespov) [reply services/docker-compose.yml]: why not? we do the same with the director?

File: services/web/server/src/simcore_service_webserver/s3.py (hunk 2 above)
Comment (pcrespov) [reply services/web/server/src/simcore_service_webserver/s3.py]: Every subsystem needs a setup and a config.  Will write down the rationale as well for future reference

**Pull request merged**